samples/Voting/Program.cs

```diff
@@ -19,12 +14,12 @@
     else
     {
         // In Kubernetes, we use environment variables and the pod manifest
-        orleansBuilder.UseKubernetesHosting();
+        //orleansBuilder.UseKubernetesHosting();
```

Comment:
Should be uncommented for !Development?

Reply:
The underlying package dependencies need to be updated to 7.0. Can't right now, this should probably be a draft.